tests/core/test_settings.py

```diff
@@ -276,27 +286,22 @@ def test_encoding_absent_in_v2(is_v1, mock_conf):
         ("core", "internal_api_url"): "http://localhost:8888",
     }
 )
-def test_get_traceback_session_if_aip_44_enabled():
+def test_get_traceback_session_if_aip_44_enabled(clear_internal_api):
     # ensure we take the database_access_isolation config
     InternalApiConfig._init_values()
     assert InternalApiConfig.get_use_internal_api() is True
 
-    # ensure that the Session object is TracebackSession
-    configure_orm()
-
-    from airflow.settings import Session
-
-    assert Session == TracebackSession
```

Comment:
removed this part of the test so we can avoid reconfigure orm

it will me an that the behavior that settings.Session is traceback session when using db isolation will be untested -- but, maybe that's acceptable